crates/core/c8y_api/src/smartrest/inventory.rs

```diff
@@ -1,7 +1,19 @@
 //! This module provides some helper functions to create SmartREST messages
 //! that can be used to create various managed objects in Cumulocity inventory.
+
+// TODO: Have different SmartREST messages be different types, so we can see
+// where these messages are used, not only created.
+//
+// TODO: both `C8yTopic::smartrest_response_topic(&EntityMetadata)` and
+// `publish_topic_from_ancestors(&[String])` produce C8y MQTT topics on which
+// smartrest messages are sent. There should be one comprehensive API for
+// generating them.
```

Comment:
This will have to be addressed by https://github.com/thin-edge/thin-edge.io/issues/2316.